scripts.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-plusplus */
+const clicked = document.querySelector('#clicked');
+const toolbar = document.querySelector('.hamburger');
+const clickedP = document.querySelector('.clicked-p');
+const navLinks = document.querySelectorAll('.clicked-ul li');
+function mobileMenu() {
+  clicked.classList = 'clicked';
+}
+function hideMobileMenu() {
+  clicked.classList = 'not-cliked';
+}
+
+toolbar.addEventListener('click', mobileMenu);
+clickedP.addEventListener('click', hideMobileMenu);
+navLinks.forEach((element) => {
+  element.addEventListener('click', hideMobileMenu);
+});
+// Creating dynamic feutured section
+
+const feautured = [
+  {
+    name: 'BIMENYIMANA Galiede',
+    title: 'MD - Managing director ',
+    descriptions:
+      'We strive to not only meet our clients expectations but exceed them',
+    image: 'images/others.jpeg',
+  },
+  {
+    name: 'RUSHEMA Anicet',
+    title: 'CTO - Chief technology officer',
+    descriptions:
+      'Company cofounder: We always do the right thing. With precision, pace and passion',
+    image: 'images/rushema.png',
+  },
+  {
+    name: 'MUSEME Amudala',
+    title: 'Developer',
+    descriptions: 'We own and shape our future',
+    image: 'images/others.jpeg',
+  },
+  {
+    name: 'NIYONSABA Gilbert',
+    title: 'Developer - backend',
+    descriptions: 'Happy coding😊',
+    image: 'images/others.jpeg',
+  },
+];
+
+// passing data into the works section
+const works = document.querySelector('.c-speakers');
+// eslint-disable-next-line no-plusplus
```

Comment:
- [ ] Nice one adding the "featured speakers" section dynamically, Please don't disable linters, you can check online about the fixes, you can check stackoverflow and it likes. It's too early to start disabling linters 😁.